proto/optimint.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+package optimint;
+
+// option go_package = "github.com/lazyledger/optimint/proto";
+import "tendermint/abci/types.proto";
+
+message Version {
+	uint32 block = 1;
+	uint32 app = 2;
+}
+
+message Header {
+	Version version = 1;
+	bytes namespace_id = 2;
+	uint64 height = 3;
+	uint64 time = 4;
+	bytes last_header_hash = 5;
+	bytes last_commit_hash = 6;
+	bytes data_hash = 7;
+	bytes consensus_hash = 8;
+	bytes app_hash = 9;
+	bytes last_results_hash = 10;
```

Comment:
We should revisit if we can't get rid of some of these hashes here too. 

ref:  
- https://github.com/tendermint/spec/issues/199
- https://github.com/tendermint/spec/issues/175

Reply:
As this is not the highest priority right now, I created https://github.com/lazyledger/optimint/issues/77.